### PR TITLE
Introduce defaultTime and onTimeChange props

### PR DESCRIPTION
**Pull request by soswow**

### CODE REVIEW

File: __tests__/index.tsx [hunk 1]
```diff
@@ -582,5 +604,36 @@ describe('VideoRenderer', () => {
...
+        buffered: {}
+      });
+      simulate(component, 'timeUpdate', {
+        currentTime: 11.5,
+        buffered: {}
+      });
+
+      expect(onTimeChange).toHaveBeenCalledTimes(2);
+      expect(onTimeChange).toHaveBeenCalledWith(10);
+      expect(onTimeChange).toHaveBeenCalledWith(11);
```
Comment (zzarcon) [inline __tests__/index.tsx:636]: Should we assert the number of times onTimeChange gets called?

File: src/video.tsx [hunk 2]
```diff
@@ -94,12 +97,20 @@ export class Video extends Component<VideoProps, VideoComponentState> {
...
   }
 
+  componentDidMount() {
+    const { defaultTime } = this.props;
+    if (this.currentElement) {
+      this.currentElement.currentTime = defaultTime;
+    }
+  }
+
   componentDidUpdate(prevProps: VideoProps) {
```
Comment (zzarcon) [inline src/video.tsx:114]: What if consumer changes the defaultTime prop? Do we want to react to that?

File: __tests__/index.tsx (hunk 1 above)
Comment (soswow) [reply __tests__/index.tsx:636]: we should. Which I do 2 lines above :-P

File: src/video.tsx (hunk 2 above)
Comment (soswow) [reply src/video.tsx:114]: how other components behave in that case? I don't think so. But I am not sure. Let's think - the only think I can do is to override current state, which is silly imo

File: src/video.tsx [hunk 3]
```diff
@@ -126,6 +137,13 @@ export class Video extends Component<VideoProps, VideoComponentState> {
 
   private onTimeUpdate = (event: SyntheticEvent<SourceElement>) => {
     const video = event.target as SourceElement;
+    const { onTimeChange } = this.props;
+
+    const flooredTime = Math.floor(video.currentTime);
```
Comment (zzarcon) [inline src/video.tsx:142]: /nit move this into the if

Comment (soswow) [reply src/video.tsx:142]: I use it in if condition

**Pull request merged**